contrib/internal/httputil/trace.go

```diff
@@ -17,30 +17,44 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
+// TraceConfig defines the configuration for request tracing.
+type TraceConfig struct {
+	Writer         http.ResponseWriter
+	Req            *http.Request
```

Comment:
```suggestion
	Request            *http.Request
```
Also, we should document each field.

Reply:
Renamed and added docs